src/sass/_shame.scss

```diff
@@ -138,3 +138,54 @@ body .row {
 .csp-inline-patch-logout {
   padding: 2em 0;
 }
+
+// /education/apply/ uses this expander button for navigation
+// Where else can I put this? Also, it could use a better name.
+.expander-button {
+  &::after {
+    background-image: url(/img/arrow-down-white.svg);
+    background-position: right bottom;
+    background-repeat: no-repeat;
+    background-size: 0.6em auto;
+    display: inline-block;
+    margin-left: 0.5em;
+    margin-right: -0.3em;
+    width: 1em;
+    height: 1em;
+    content: ''; // Add content for accessibility purposes?
```

Comment:
I think it might be a combination of that and running an automated axe test on it. We're talking more about how to run our 508 tests on Friday so I think we can address this then.